services/api/.env.example

```
@@ -19,6 +19,18 @@ ENABLE_AUTOMIGRATE=false
 # NOTE: not yet wired — field exists in config but main.go always starts scheduler (scope: A5-2)
 ENABLE_SCHEDULER=true
 
+# ── Tracing / OTLP ────────────────────────────────────────────────────────────
+# Vendor-neutral tracing baseline. Disabled by default; staging/production must
+# set an OTLP traces endpoint before enabling.
+TRACING_ENABLED=false
+OTEL_SERVICE_NAME=tachigo-api
+OTEL_ENVIRONMENT=development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The OTEL_ENVIRONMENT key should go before the OTEL_SERVICE_NAME key

(UnorderedKey)